### PR TITLE
Journalist can add images to article 

### PR DESCRIPTION
**Pull request by pierre-1**
# [PT Story](https://www.pivotaltracker.com/story/show/171811139)

## User Story
```
As a journalist 
In order to make my articles more visually appealing 
I would like to be able to upload and display images to my articles
```
## Changes proposed in this pull request:
* Adds image upload functionality for journalists when they create articles
* Tests for journalist being able to click upload 

## What I have learned working on this feature:
* How to use react-images-uploading yarn package 

## Packages/Gems added
- React-Images-Uploading

## Screenshots (Only if client side story)
Screenshot from 2020-03-24 17-20-37
## Lo-Fi (Only if client side story)
Pasted Image at 2020_03_16_07_59

### CODE REVIEW

File: src/components/CreateArticle.jsx [hunk 1]
```diff
@@ -67,6 +78,36 @@ class CreateArticle extends Component {
             name="category"
             width={6}
           />
+          {/* <Form.Input id="image" type="file" name="image" key="image"/> */}
```
Comment (emtalen) [inline src/components/CreateArticle.jsx]: Remove comments

File: src/components/CreateArticle.jsx [hunk 2]
```diff
@@ -67,6 +78,36 @@ class CreateArticle extends Component {
...
+          <ImageUploading onChange={this.onImageDropHandler}>
+            {({ imageList, onImageUpload }) => (
+              // write your building UI
+              <div className="upload__image-wrapper">
+                <Button id="imageUpload" onClick={onImageUpload}>
+                  Upload images
+                </Button>
+                &nbsp;
```
Comment (emtalen) [inline src/components/CreateArticle.jsx:87]: What is this doing?

**Review by emtalen — COMMENTED** [review]
I think we should wait to merge it before we can test the uploading of the image correctly and meanwhile work on other features, otherwise good job! 💯

File: src/components/CreateArticle.jsx (hunk 2 above)
Comment (pgaunitz) [reply src/components/CreateArticle.jsx:87]: not a typo. supposed to be there

File: src/components/CreateArticle.jsx (hunk 1 above)
Comment (pgaunitz) [reply src/components/CreateArticle.jsx]: fixed

**Review by emtalen — APPROVED** [review]

**Review by kfrostare — APPROVED** [review]
Updated the title so it's consistent with the PT, otherwise looks good /KF

**Pull request merged**